deploying-a-django-app-on-okteto.md

```diff
@@ -0,0 +1,704 @@
+# Deploying a Django App on Okteto
+
+## Introduction
+
+In this tutorial, We will building and deploying a Django Role based Authorization Service API(RBAS) on Okteto.
+
+An RBAS application, is a system with checks to give access to users based on their roles in the system. 
+
+What is Okteto ?
+
+Okteto is a cloud platform that enables developers develop and deploy cloud like Docker and Kubernetes applications. 
+
+What is Django?
+
+Django is one of the web framework of python for building APIs. Django is a framework that comes with out of the box functionality used in developing web apps and API, making your development faster and easier.
+
+
+## Prerequisites 
+- Knowledge of building APIs
+- Basic Knowledge of Django
+- Basic Knowledge of Cloud Infrastructure 
+
+## Scaffold Project
+
+To start building, we will first create and activate a virtual environment for our project:
+
+
+    $ python3 -m venv venv
+    $ source venv/bin/activate
+
+Next, we install Django:
+
+
+    $ pip install django
+
+Running `django-admin`. 
+
+
+    $ django-admin startproject simple_rbas
+
+`django-admin` is a command provided by Django to help bootstrap your project, creating essential directories and files to get started.
+
+Navigate into the new project created;
+
+
+    $ cd simple_rbas
+
+Create `requirements.txt` file in the root folder and add the following packages:
+
+
+    django
+    djangorestframework
+    django-environ
+    psycopg2-binary
+    gunicorn
+
+Install the packages using the `pip` command:
+
+
+    pip install -r requirements.txt 
```

Comment:
could you add a final paragraph for this section explaining what did we accomplish so far?